packages/gatsby-source-wordpress/src/normalize.js

```diff
@@ -138,11 +138,15 @@ exports.liftRenderedField = entities =>
 
 // Exclude entities of unknown shape
 exports.excludeUnknownEntities = entities =>
-  entities.filter(e => e.wordpress_id) // Excluding entities without ID
+  entities.filter(e => e.wordpress_id || e.__type === `wordpress__wp_settings`) // Excluding entities without ID, or WP Settings
 
 exports.createGatsbyIds = (createNodeId, entities) =>
   entities.map(e => {
-    e.id = createNodeId(`${e.__type}-${e.wordpress_id.toString()}`)
+    if (e.wordpress_id) {
```

Comment:
here would be good idea to put comment mentioning `excludeUnknownEntities` and why we have to ways of creating node id